src/language/api/verificationGutterStatusParams.ts

```diff
@@ -0,0 +1,77 @@
+import { DocumentUri, integer } from 'vscode-languageclient';
+
+export interface IVerificationDiagnosticsParams {
```

Comment:
I would include `Gutter` in this name and leave out `Diagnostics` (since it refers to LSP's diagnostics)